release: v1.41.0 (#4671)
apollographql/router: .changesets/fix_geal_subgraph_streaming_compression.md

### Implement streaming compression for subgraph requests ([Issue #4648](https://github.com/apollographql/router/issues/4648))

> This fixe subgraph HTTP requests to compress the body in streaming instead of loading it entirely in the compression engine before sending everything at once. This reuses the compression layer that the router uses to compress client responses.

Reviewer: ```suggestion
The router now compresses the body of subgraph HTTP requests instead of loading them entirely in the compression engine before sending everything at once. This fix reuses the compression layer that compresses client responses.
```

Reviewer 2: applied in https://github.com/apollographql/router/commit/16568f457761a019cd2c745c0ec2d4143c4cc1e4
